src/layout/OrgLayout.tsx

```diff
@@ -10,10 +11,55 @@ const OrgLayout = ({ children, orgId } : OrgLayoutProps) : JSX.Element => {
     const orgQuery = useQuery(['org', orgId], getOrg(orgId));
 
     return (
-        <div>
-            <h1>{ orgQuery.data.title }</h1>
+        <>
+            <figure style={{
+                alignItems: 'flex-end',
+                background: 'url(/cover.jpg)',
+                backgroundPosition: 'center',
+                backgroundSize: 'cover',
+                boxSizing: 'border-box',
+                display: 'flex',
+                height: '500px',
+                justifyContent: 'flex-end',
+                margin: '0',
+                padding: '1rem',
+                width: '100%'
+            }}>
+                <Link href='/'>
+                    <a style={{ color: 'white' }}>Edit Page</a>
+                </Link>
+                <button style={{ marginLeft: '20px' }}>Unfollow</button>
+            </figure>
+            <img
+                alt='Organization avatar'
+                src={ `https://api.zetk.in/v1/orgs/${orgId}/avatar` }
+                style={{
+                    height: 'auto',
+                    left: '1rem',
+                    position: 'absolute',
+                    top: '520px',
```

Comment:
This position currently is relative to the top of the page, which means that if the size of the main header (from `DefaultLayout`) changes, this goes out of alignment.

This is suboptimal because it means that the `OrgLayout` component is not self-contained.

I don't know if it's necessary to rely on absolute positioning here?

But regardless of the absolute positioning, I think it would make sense to wrap the "organization header", i.e. most of this component, in a div or something like that, and make that `position: relative`.

Reply:
Yes, that's true! I added a section with `position: relative`. Left the absolute in there but can change it if you have something else in mind.